stdlib/ssl.pyi

```diff
@@ -103,25 +103,48 @@ if sys.platform == "win32":
     def enum_certificates(store_name: str) -> _EnumRetType: ...
     def enum_crls(store_name: str) -> _EnumRetType: ...
 
-CERT_NONE: int
-CERT_OPTIONAL: int
-CERT_REQUIRED: int
-
-VERIFY_DEFAULT: int
-VERIFY_CRL_CHECK_LEAF: int
-VERIFY_CRL_CHECK_CHAIN: int
-VERIFY_X509_STRICT: int
-VERIFY_X509_TRUSTED_FIRST: int
-
-PROTOCOL_SSLv23: int
-PROTOCOL_SSLv2: int
-PROTOCOL_SSLv3: int
-PROTOCOL_TLSv1: int
-PROTOCOL_TLSv1_1: int
-PROTOCOL_TLSv1_2: int
-PROTOCOL_TLS: int
-PROTOCOL_TLS_CLIENT: int
-PROTOCOL_TLS_SERVER: int
+class VerifyMode(enum.IntEnum):
+    CERT_NONE: int
+    CERT_OPTIONAL: int
+    CERT_REQUIRED: int
+
+CERT_NONE: VerifyMode
+CERT_OPTIONAL: VerifyMode
+CERT_REQUIRED: VerifyMode
+
+class VerifyFlags(enum.IntFlag):
+    VERIFY_DEFAULT: int
+    VERIFY_CRL_CHECK_LEAF: int
+    VERIFY_CRL_CHECK_CHAIN: int
+    VERIFY_X509_STRICT: int
+    VERIFY_X509_TRUSTED_FIRST: int
+
+VERIFY_DEFAULT: VerifyFlags
+VERIFY_CRL_CHECK_LEAF: VerifyFlags
+VERIFY_CRL_CHECK_CHAIN: VerifyFlags
+VERIFY_X509_STRICT: VerifyFlags
+VERIFY_X509_TRUSTED_FIRST: VerifyFlags
+
+class _SSLMethod(enum.IntEnum):
+    PROTOCOL_SSLv23: int
```

Comment:
This one seems to be special-cased somehow: https://github.com/python/cpython/blob/main/Lib/ssl.py#L125 I think it should be omitted from the enum.

Reply:
Yeah it is a special case, because it is equivalent to `PROTOCOL_TLS`
https://github.com/python/cpython/blob/0a8ae8a50a0fea3d39ec49b220a5c7a5b70e36f8/Lib/ssl.py#L153

Reply:
Right, so I think it's not actually in the enum?

Reply:
yeah, I think you are right. `PROTOCOL_SSLv23` is just monkey patched on it. Should I remove it then?

Reply:
Well, it depends on whether it exists in the enum at runtime (i.e., does `ssl._SSLMethod.PROTOCOL_SSLv23` work).

Reply:
yepp, it does
```python
>>> import ssl
>>> print(ssl._SSLMethod.PROTOCOL_SSLv23)
_SSLMethod.PROTOCOL_TLS
>>> print(ssl._SSLMethod.PROTOCOL_SSLv23.value)
2
```

Reply:
OK, then let's just keep it as is